### PR TITLE
class.md created

### PR DESCRIPTION
**Pull request by Cesarcas1958**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

I have created a file class.md as requested

### CODE REVIEW

File: content/html/concepts/attributes/terms/class/class.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Classes'
+
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:3]:
```suggestion
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Definition: 'Classes are HTML attributes used to select one or more elements for CSS styling or JavaScript purposes. The class attribute is commonly used for assigning styles to one or more elements. Inside the stylesheet, a class name is selected with the following syntax: css .class-name { }  A class selector is a name preceded by a period (.).'
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:4]:
```suggestion
Description: 'Classes are HTML attributes used to select one or more elements for the application of styling or logic.'
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:5]:
```suggestion
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:9]:
```suggestion

```

File: content/html/concepts/attributes/terms/class/class.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+Tags:
+  - 'Attributes'
+  - 'Classes'
+  - 'Style'
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:13]:
```suggestion
  - 'Style'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---
```

Comment (caupolicandiaz) [reply content/html/concepts/attributes/terms/class/class.md:13]: The formatting for the metadata section has several errors. Be sure to always use one of our templates or another entry as a starting point, you should never create an entry from scratch.

Additionally, you should never just copy a paste the written content from another source, even another Codecademy source. All entries should be original, which means you have to phrase the concept and other elements in your own words.

File: content/html/concepts/attributes/terms/class/class.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+Classes are HTML attributes used to select one or more elements for CSS styling or JavaScript purposes.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:15]:
```suggestion
Classes are HTML attributes used to select one or more elements for the application of styling or logic. The class attribute is commonly used inside a stylesheet for the application of specific styling such as setting color and position parameters for elements on a page. Additionally, the class attribute is also used to reference elements relative to actions or interactivity that are implemented in the Javascript file.
```

File: content/html/concepts/attributes/terms/class/class.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+      <h2 class="sub-headline">_</h2>
+
+      <h2 class="sub-headline">_</h2>
+    </div>
+  </body>
+</html>
+```
+
+![Screenshot of styling for heading and sub-headings.](https://i.imgur.com/fMWV8DY.png)
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:62]: Replace this example with an original of your own design

File: content/html/concepts/attributes/terms/class/class.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+  for (let i = 0; i < subheadlines.length; i++) {
+    subheadlines.item(i).innerText = `Sub-headline ${i + 1}`;
+  }
+};
+```
+
+The code above is assigning an event handler to the `onload` property of the global `window` object in the browser. When the window as loaded, the elements with `class=sub-headilne` are collected in `subheadlines`. They are then looped over, their `innerText` property being changed each iteration.
+
+![Screenshot of selecting heading and sub-headings with JavaScript](https://i.imgur.com/2uIJMAq.png)
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/class/class.md:82]: Again replace this with original content

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Cesarcas1958 I've finished an initial review. This entry is quite rough, I would ask that you please review our contribution guidelines and my notes carefully. And then make the appropriate changes. If you have any questions about what is being requested please reach out. Thanks

[https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md](url)

**Comment by Cesarcas1958** [comment]
Por tu nombre asumo que hablas español y de ser asi me pudieras enviar esos
detalles en español gracias. Si no es asi hazmelo saber
Gracias

**Comment by Cesarcas1958** [comment]
I made changes in class.md file as requested but I am having trouble to push the changes. Could you give some advice?
This is the message I am getting:
PS C:\Users\César Castellanos\Documents\GitHub\docs\content\html\concepts\attributes\terms\class> git pull origin new1
fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Pull request closed without merging**